lib/spoom/deadcode/indexer.rb

```diff
@@ -330,62 +217,6 @@ def reference_method(name, node)
         @index.reference(Reference.new(name: name, kind: Reference::Kind::Method, location: node_location(node)))
       end
 
-      # Context
-
-      sig { returns(Prism::Node) }
-      def current_node
```

Comment:
Were these methods unused? There are some definitions that are removed without any callsites being removed.

Reply:
They were helpers for the plugins to access more context around the node it was analyzing. All this knowledge is now moved inside the `Model` so plugins can access it more easily.